### DISCUSSION TITLE
.net 8 SR1 what does it mean in terms of date

### DISCUSSION THREAD

**Discussion by developer9969** [dotnet/maui#17734] [General]
Hi many bugs have been moved to "SR1" 

What does it mean in terms of delivery? 
Some of them are critical in my view.

anyone ?

**Comment by HuaFangYun** [comment]
What does SR mean?

**Reply by sjordanGSS** [reply]
Service Release

**Comment by FM1973** [comment]
I think we will first get RC2. Then the official release (GA) and then SR1. Maybe at the end of the year? Anyone from the MAUI team to let us know how this is planned? And then... will the "we do everything new in .net 8 and maybe backport it to .net 7" happen again for .net 8 and 9? 8 is a long term release. this should be different then!

**Reply by kcrg** [reply]
In MAUI world there is no long term release, every major release have one and a half year long support.
https://dotnet.microsoft.com/en-us/platform/support/policy/maui

**Reply by AndreasReitberger** [reply]
There, Microsoft says, the update on a monthly cycle. However, the last Net7 release is Juli 2023. So Microsoft doesn't work after their own guidelines, sadly.

> Servicing updates are made available for supported versions of .NET MAUI. Servicing updates ship on the same monthly update cycle as the rest of .NET — in months where there are fixes, the update ships on the second Tuesday of the month (also known as Patch Tuesday) along with updates for .NET.
> 
> .NET MAUI updates may include a mix of security and non-security fixes in a single update.

**Reply by MAUIoxo** [reply]
Where can we get these monthly updates then? Right now, I am working with nightly to see if certain things are fixed. But that's also with some ups and downs like the last kick from `<MauiVersion>8.0.6-nightly.9880</MauiVersion>` to the next higher versions with breaking things included so that my App won't build anymore (see [https://github.com/dotnet/maui/discussions/20093](https://github.com/dotnet/maui/discussions/20093)

**Comment by developer9969** [comment]
It would be nice if @davidortinau  could let us know what the SR1 plan for release is, and I hope id does not become another backlog place where bugs never get fixed.

**Comment by HuaFangYun** [comment]
SR1,SR2,SR3 Maybe I don't understand the complexity of maui, but I want maui to speed up the repair of bug and have a better optimization plan and team.

**Comment by AndreasReitberger** [comment]
Also looking forward for a SR (Net8), the NET 8 RC is out, but not usable for any of my MAUI apps due to a lot of issues keeping me from building the application or running it. Most of them seemed to be fixed already, but there is no new nightly, nor the next SR update in sight... Fixing bugs in NET7 seems to be a way down on the ToDo list, till NET 8 is GA (just assuming).
So hopefully the next NET8 release will make it usable.

**Comment by jurganson** [comment]
Any news on this SR1 release and when it will be available through .NET 8 coming with Visual Studio 2022?

**Comment by jkommeren** [comment]
> Any news on this SR1 release and when it will be available through .NET 8 coming with Visual Studio 2022?

I saw a reference to a version 8.0.5 somewhere in bug reporting PR. We're now on 8.0.3. I'm guessing that'll be the next version? (=SR1?)

Here's the pr:
https://github.com/dotnet/maui/pull/19795

Also this PR seems to suggest 8.0.5 will be stable? https://github.com/dotnet/maui/pull/19693

I think the S is for Servicing. R is release?

**Comment by asi-evin** [comment]
I see the milestone for SR1 is marked as 100% complete. Does that mean it was released, or soon to be released?

**Reply by Pastajello** [reply]
we will find out veeeery soon from the looks of it:

https://github.com/dotnet/maui/pull/19936

but dang, they could at least say something :(

**Reply by NonameMissingNo** [reply]
Today! Woot woot
